lib/containers/pr-changed-files-container.js

```diff
@@ -99,6 +106,13 @@ export default class PullRequestChangedFilesContainer extends React.Component {
       if (response && response.ok) {
         const rawDiff = await response.text();
         const multiFilePatch = this.buildPatch(rawDiff);
+
+        this.mfpSubs.dispose();
+        this.mfpSubs = new CompositeDisposable();
+        for (const fp of multiFilePatch.getFilePatches()) {
+          this.mfpSubs.add(fp.onDidChangeRenderStatus(() => this.forceUpdate()));
```

Comment:
These `.forceUpdate()` calls are kind of a smell to me. I don't think we can do better right now without some serious restructuring of our state management, though.